mealpy.py

```diff
@@ -121,14 +128,72 @@ def cancel_current_meal(self):
         raise NotImplementedError()
 
 
-SCHEDULER = BlockingScheduler()
+def get_mealpal_credentials():
```

Comment:
This is an inaccurate name, it returns a `MealPal` object.

Reply:
You're right, I was having trouble coming up with a good name because this function had multiple responsibilities. Extracting out the object creation part makes more sense.